internal/queries/queries.sql

```diff
@@ -25,8 +29,8 @@ LEFT JOIN
     pg_catalog.pg_namespace AS parent_namespace
     ON parent_c.relnamespace = parent_namespace.oid
 WHERE
-    c.relnamespace
-    = (SELECT oid FROM pg_catalog.pg_namespace WHERE nspname = 'public')
+    table_namespace.nspname NOT IN ('pg_catalog', 'information_schema')
```

Comment:
This is the same filter Postgres uses in its pg_stat_user_tables view to filter down from pg_stat_all_tables.